pubspec.yaml

```diff
@@ -44,6 +50,9 @@ flutter:
   # the material Icons class.
   uses-material-design: true
 
+  assets:
```

Comment:
You can decide whether to specify each file on a new line, or just make the reference to `assets/images/` and then all the contents will be added to assets, just a reminder 👍

Reply:
done